packages/utils/util.ts

```diff
@@ -51,7 +51,7 @@ export function getPropByPath(obj: any, path: string, strict: boolean): {
 
   const keyArr = path.split('.')
   let i = 0
-  for (i; i < keyArr.length - 1; i++) {
+  for (const len = keyArr.length; i < len - 1; i++) {
```

Comment:
Do you think this modification means anything? Was it improving any readability?

Reply:
Personal habits, and same with v2 version， i can revert this